qfieldcloud_sdk/sdk.py

```diff
@@ -1395,6 +1419,193 @@ def patch_organization_members(
+        """Gets a list of organization teams.
+
+        Args:
+            organization: The name of the organization.
+
+        Returns:
+            A list of TeamModel objects representing the organization's teams.
+
+        Example:
+            ```python
+            teams = client.get_organization_teams(organization="My_Organization_Clan")
+            for team in teams:
+                print(team.team)
+            ```
+        """
+        teams = cast(
+            List[TeamModel],
+            self._request_json("GET", f"organizations/{organization}/teams/"),
+        )
+        return teams
+
+    def create_organization_team(self, organization: str, team_name: str) -> TeamModel:
+        """Creates a new organization team.
+
+        Args:
+            organization: The name of the organization.
+            team_name: The name of the new team.
+
+        Returns:
+            A TeamModel object representing the newly created team.
+
+        Example:
+            ```python
+            client.create_organization_team(organization="My_Organization_Clan", team_name="surveyors")
+            ```
+        """
+        team = cast(
+            TeamModel,
+            self._request_json(
+                "POST",
+                f"organizations/{organization}/teams/",
+                {"team": team_name},
+            ),
+        )
+        return team
+
+    def get_organization_team(self, organization: str, team_name: str) -> TeamModel:
+        """Gets an organization team by name.
+
+        Args:
+            organization: The name of the organization.
+            team_name: The name of the team to retrieve.
+
+        Returns:
+            A TeamModel object representing the requested team.
+
+        Example:
+            ```python
+            client.get_organization_team(organization="My_Organization_Clan", team_name="surveyors")
+            ```
+        """
+        team = cast(
+            TeamModel,
+            self._request_json(
+                "GET", f"organizations/{organization}/teams/{team_name}/"
+            ),
+        )
+        return team
+
+    def patch_organization_team(
+        self, organization: str, team_name: str, new_team_name: str
+    ) -> TeamModel:
+        """Patches an organization team.
+
+        Args:
+            organization: The name of the organization.
+            team_name: The current name of the team to modify.
+            new_team_name: The new name for the team.
+
+        Returns:
+            A TeamModel object representing the updated team.
+
+        Example:
+            ```python
+            client.patch_organization_team(organization="My_Organization_Clan", team_name="surveyors", new_team_name="field_surveyors")
+            ```
+        """
+        team = cast(
+            TeamModel,
+            self._request_json(
+                "PUT",
+                f"organizations/{organization}/teams/{team_name}/",
+                {"team": new_team_name},
```

Comment:
```suggestion
                {
                    "team": new_team_name,
                },
```

Prefer multiline dicts.